sdk/python/test_data/pipelines/pipeline_with_condition.py

```diff
@@ -36,7 +36,7 @@ def my_pipeline(text: str = 'condition test'):
     flip1 = flip_coin_op().set_caching_options(False)
     print_op(msg=flip1.output)
 
-    with dsl.Condition(flip1.output == 'heads'):
```

Comment:
Would there be some duplication between these the if/elif/else samples we might already have? Consider reducing the duplication.